src/test/java/co/rsk/federate/BtcToRskClientTest.java

```diff
@@ -103,7 +104,7 @@ void setup() throws PeginInstructionsException, IOException {
         when(activationConfig.isActive(eq(ConsensusRule.RSKIP89), anyLong())).thenReturn(true);
 
         bridgeRegTestConstants = BridgeRegTestConstants.getInstance();
-        genesisFederation = bridgeRegTestConstants.getGenesisFederation();
+        activeFederation = TestUtils.createFederation(bridgeRegTestConstants.getBtcParams(),1);
```

Comment:
A federation with just 1 member is not very realistic. Maybe use 9 as the one currently in mainnet

Reply:
For this test, reverted `activeFederation` to `genesisFederation` again since I noticed when I tried to apply a number of federation members greater than 1, it didn't work. This test is tied to genesisFederation, however, I applied for the rest of the tests, 9 as the number of federation members.

**Changes included in the last commit**
	- Setting 9 as the number of federation members to make the tests more realistic.
	- Reverted activeFederation to genesisFederation again in some tests, since some of them are tied to genesisFederation.
	- Adding getGenesisFederation Method to TestUtils.
	- Improving TestUtils by adding the final keyword at the class level and a private constructor.

Reply:
but still... the genesisFederation in regtest has three members, not one.
Probably the best idea is to understand why is failing with one member :)

Reply:
@marcos-iov @julia-zack Done!